src/components/z-status-tag/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { Component, Prop, h, Host } from "@stencil/core";
+import { StatusTagVariant, StatusTagType } from "../../beans";
+import classNames from "classnames";
+
+@Component({
+  tag: "z-status-tag",
+  styleUrl: "styles.css",
+  shadow: false,
```

Comment:
why this?

Reply:
Lasciando **shadow: true** sembra non si possa utilizzare il tag che si scrive e si deve per forza aggiungere un elemento che contenga testo e icona con le proprietà adesso assegnate allo z-status-tag